app/controllers/case_contacts_controller.rb

```diff
@@ -134,6 +136,17 @@ def restore
     redirect_to request.referer
   end
 
+  def case_contacts_creation_times_in_last_week
+    # Get the case contacts created in the last week
+    case_contacts = CaseContact.where("created_at >= ?", 10.week.ago)
```

Comment:
metrics are much more valuable per org- are we planning to split these out per org in a later PR?

Reply:
No these are global metrics